samples/Shell/Fabimals/Droid/MainActivity.fs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Droid
+
+open System
+open Android.App
+open Android.Content
+open Android.Content.PM
+open Android.Runtime
+open Android.Views
+open Android.Widget
+open Android.OS
+open Xamarin.Forms.Platform.Android
+
+[<Activity(Label = "Fabimals", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = (ConfigChanges.ScreenSize ||| ConfigChanges.Orientation))>]
+type MainActivity() =
+    inherit FormsAppCompatActivity()
+    override this.OnCreate(bundle : Bundle) =
+        FormsAppCompatActivity.TabLayoutResource <- Resources.Layout.Tabbar
```

Comment:
We should point out that Toolbar/TabLayoutResource is needed.

Reply:
Is it?
That's interesting. Maybe that's why the Android app wasn't working at first.
I'll try without, see what kinds of issue it raises.

Reply:
AFAIK yes but would be greta if you could test this.

Reply:
Doesn't seem to be a requirement. Just tried with those 2 lines commented out and Fabimals just works the same.